### ISSUE TITLE
Future plan? 

### ISSUE THREAD

**Issue by Iknowyou808** [citizenfx/fivem#1428]
As you know, Rockstar announced a new part of GTA, and I have a question for the fivem developers, do you think we will get new formats (probably even more difficult than rdr 2), will you support modding, study formats, as I think that modding New gta  will be even more difficult than rdr 2, and we will get the same slow development as in rdr 2 (yes, few people play it, but it's still a game).  I've been told by some people that it's possible that future gta modding will evolve faster than gta 5 and rdr 2, what do you think about that?.

**Comment by Ekinoxx0** [comment]
Is this the same guy doing speculation on an issue as usual ?
FiveM code is as future proof as possible on current knowledge.

FiveM don't do speculation on future format or anything that Rockstar will do. They work on things when we have information, there is no point in speculating on something we don't have any information at all.

If you want to discuss this kind of topic, Discord or the forum are better suited. This is reserved for actionable issue and not questions.

**Comment by aiden-z-source** [comment]
> We’re certain this is a burning question for all of you: Will there be a SixM? The answer is a little complicated for many reasons. However, assuming there will be a PC port, we cannot predict what the game is like, or how complex the systems are before we make any decisions. Only time will tell!

The [Cfx.re Monthly Update - February 2022](https://forum.cfx.re/t/cfx-re-monthly-update-february-2022-edition/4795964?u=aiden_z) edition discusses this, also reading all these monthly updates is very informational so I suggest doing so :P

**Comment by Iknowyou808** [comment]
> Is this the same guy doing speculation on future gta as usual ? FiveM code is as future proof as possible on current knowledge.
> 
> FiveM don't do speculation on future format or anything that Rockstar will do. They work on things when we have information, there is no point in speculating on something we don't have any information at all.
> 
> If you want to discuss this kind of topic, Discord or the forum are better suited. This is reserved for actionable issue and not questions.

Sometimes I want to think about it, I'm just really scared that modding the new gta will be even more difficult and slower to develop as it was with rdr 2 and gta 5 at the beginning, we got animations after 5 years, I think in new gta we will get it even later. I don't want to think so, but it looks like it will be.

**Comment by Iknowyou808** [comment]
> > We’re certain this is a burning question for all of you: Will there be a SixM? The answer is a little complicated for many reasons. However, assuming there will be a PC port, we cannot predict what the game is like, or how complex the systems are before we make any decisions. Only time will tell!
> 
> The [Cfx.re Monthly Update - February 2022](https://forum.cfx.re/t/cfx-re-monthly-update-february-2022-edition/4795964?u=aiden_z) edition discusses this, also reading all these monthly updates is very informational so I suggest doing so :P

Yes I read this, but still decided to write again what I write every month, maybe more, I wonder what other people think, I still hope that the future of modding will be better than it was in gta 5 / rdr 2 and it will develop faster

**Comment by aiden-z-source** [comment]
> but still decided to write again what I write every month

Apparently, they know they are doing this too lol, fun.
